### DISCUSSION TITLE
It doesn't let me connect.

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#1961] [Help]
Hello I'm new to Minestom. And it says **_finishConnect(...)  failed: Connection refused_** when try to join from server list and won't work. but it works when I connect from LAN section in server list. Could anyone help me fix this?

**Comment by zax71** [comment]
Could you show the server logs and the code surrounding your `MinecraftServer#start` method

**Reply by ghost** [reply]
https://github.com/Minestom/Minestom/assets/114764181/7d0d7a68-7928-4aef-91a2-5d1f26999e2d

![Screenshot from 2023-11-04 23-16-17](https://github.com/Minestom/Minestom/assets/114764181/5477d10e-d525-43f7-a7e2-aa81d4e3bb41)
![Screenshot from 2023-11-04 23-16-17](https://github.com/Minestom/Minestom/assets/114764181/7bd8c306-99a3-4d17-9ffa-34a2da73779e)
that what it says

**Reply by zax71** [reply]
Could you send the code in your main class as well. Use https://paste.gg if you’re not planning to upload this to GitHub

**Reply by ghost** [reply]
https://paste.gg/p/anonymous/bf9f7386abf94bf4a95aa008ea989ab2
